### ISSUE TITLE
Dev Console SuperchainERC20 Support

### ISSUE THREAD

**Issue by fainashalts** [ethereum-optimism/ecosystem#505]
This parent issue is for the design and implementation of the console frontend to support SuperchainERC20 leading up to Devcon and includes links to static resources such as documentation and marketing content. 

A design exists [here](https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2-Hi-Fi?node-id=5592-18542&node-type=canvas&t=jDKPXLhGJGPgLEea-0) but please confer with Mayank for finalization and questions.

**Comment by mayanksayshi** [comment]
Hey @fainashalts ,

Regarding the SuperchainERC20 Dev Console ticket, I see that it covers design and implementation leading up to DevCon. Could you clarify which elements of the console are considered P0? I want to make sure I prioritize the right features and links to documentation and marketing content.

**Comment by mayanksayshi** [comment]
Here is the link to the designs for the Dev console. @jvmi7 

https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2-Hi-Fi?node-id=5768-12656

**Comment by fainashalts** [comment]
@jvmi7 @tarunkhasnavis please link any PRs related to this work in this issue so we can use it as a source of truth on progress. Thank you!

**Comment by tarunkhasnavis** [comment]
Made progress on 
- Updating home page 
- Implementing Build Page (starterkit info)

Also made some design updates along the way and moving to App templates page

**Comment by mayanksayshi** [comment]
Hey @tarunkhasnavis, we made a minor update to the dev console homepage. We realised Supersim needed its own card, so we added that. We also moved the Superchain safe under launch and grow. You can find the updated design here.

https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2?node-id=5768-12977&t=WfB9u7CbeBDQRih2-4

Please let me know if you have any questions.

**Comment by tarunkhasnavis** [comment]
okay, I captured all of this work in the Eng tracking issue https://github.com/ethereum-optimism/ecosystem/issues/542

the 3 main issues being here:
- https://github.com/ethereum-optimism/ecosystem/issues/554
- https://github.com/ethereum-optimism/ecosystem/issues/555
- https://github.com/ethereum-optimism/ecosystem/issues/552

**Comment by mayanksayshi** [comment]
Actually, there is one more thing this [image](https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2?node-id=5893-13470&t=WfB9u7CbeBDQRih2-4) is still work in progress I am waiting on some feedback from the creative team and it's taking longer than expected. I would request that you please hide it from the frontend for now. Thank you.

**Comment by tarunkhasnavis** [comment]
Sounds good! The changes are hidden behaving a feature flag so I can just swap out the image when you have it

**Comment by mayanksayshi** [comment]
Hey @tarunkhasnavis 

Another small change we need to make is to update the logo train according to Amanda's marketing guidelines. 
![image](https://github.com/user-attachments/assets/488cddcd-63a3-4145-a29e-b03ad0de139a)

https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2?node-id=5768-12987&t=oiHsc6CyMtTFQHZt-4


The GIF for the Superchain ERC20 Starterkit is also ready. referring to this [comment](https://github.com/ethereum-optimism/ecosystem/issues/505#issuecomment-2447959310) 

Here is the Gif 
![Asset](https://github.com/user-attachments/assets/fffa012a-7188-4742-a1b9-f5f8c15524f9)


Please let me know if you have any questions :)

**Comment by mayanksayshi** [comment]
@tarunkhasnavis, please update the copy of the SuperchainERC20 Starter kit card on the home page.

Currently, it's this.
![image](https://github.com/user-attachments/assets/9c515dd4-3db9-4d7f-8b59-eb4bd9a34016)

New Copy  👇
![image](https://github.com/user-attachments/assets/cbddb4c2-0dd1-43c0-83aa-7b54d3d786b6)

Figma link https://www.figma.com/design/JKWr4DO8BMxAB5r7qYvbIs/Dev-Console-1.2?node-id=5812-3274&t=tiFQ08M4xVy7qfsY-4

Thank you :)

**Comment by tarunkhasnavis** [comment]
@mayanksayshi 

1. Can I get the GIF in a downloadable format. I am unable to use it at the moment. 
2. Just confirming that we are removing the `Paymaster` but we are replacing it with the `Testnet Paymaster` card (check figma)

Added these comments in Figma as well

**Comment by mayanksayshi** [comment]
> @mayanksayshi
> 
>     1. Can I get the GIF in a downloadable format. I am unable to use it at the moment.
> 
>     2. Just confirming that we are removing the `Paymaster` but we are replacing it with the `Testnet Paymaster` card (check figma)
> 
> 
> Added these comments in Figma as well

Hey @tarunkhasnavis we are removing all cards related to the Paymaster. 

https://optimismfoundation.slack.com/archives/C050Z46S3QU/p1730822760075819?thread_ts=1730813146.919099&cid=C050Z46S3QU

I have updated the designs in Figma as well. :)

**Comment by tarunkhasnavis** [comment]
All eng tasks are completed now: https://github.com/ethereum-optimism/ecosystem/issues/542

We can see what comes out of the [bug bash](https://www.notion.so/oplabs/Bug-Bash-Dev-Console-Multichain-Faucet-135f153ee162808ba1d0e28164640390) for more changes to be fully complete
